src/App.js

```diff
@@ -1,11 +1,21 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+// import React from 'React';
+import Header from '/Header.js';
+import Main from 'Main.js';
+import Footer from '/Footer.js';
 
 function App() {
   return (
-    <div className="App">
-      <header><h1>Horned Beast</h1>
-      </header>
+    // // <div className="App">
+    //   <header><h1>Horned Beast</h1>
+    //   </header>
+    // </div>
+    <div>
+      <Header />
+      <Main />
+      <Footer />
+
     </div>
   );
 }
```

Comment:
Hello there,
  Make sure to remove commented out code before merge (if you are using comments to help guide you, no worries, if you are commenting code to describe something then cool, but if it is no longer in use, poof! be gone :) )